content/en/blog/2025/ai-agent-observability/index.md

```diff
@@ -0,0 +1,225 @@
+evolution comes the critical need for AI Agent Observability - especially when
+scaling these agents to meet enterprise needs. Without proper monitoring,
+tracing, and logging mechanisms, diagnosing issues, improving efficiency, and
+ensuring reliability in AI Agent-driven applications will be challenging.
+
+### What is an AI Agent
+
+![Sample RAG based application w/ReAct reasoning/planning](ai-agent.png)
+
+An AI agent is an application that uses a combination of a LLM capabilities,
+tools to connect to the external world, and high-level reasoning to achieve a
+desired end goal or state; On the other hand, agent can also be treated as
+systems where LLMs dynamically direct their own processes and tool usage,
+maintaining control over how they accomplish tasks.
+
+You can also refer to following to get more detail for AI agents:
+
+- [Google: What is an AI agent?](https://cloud.google.com/discover/what-are-ai-agents)
+- [IBM: What are AI agents?](https://www.ibm.com/think/topics/ai-agents)
+- [MicroSoft: AI agents — what they are, and how they’ll change the way we work](https://news.microsoft.com/source/features/ai/ai-agents-what-they-are-and-how-theyll-change-the-way-we-work/)
+- [AWS: What are AI Agents?](https://aws.amazon.com/what-is/ai-agents/)
+- [AWS: Building effective agents](https://www.anthropic.com/research/building-effective-agents)
+
+### Observability and Beyond
+
+Typically, telemetry from applications is used to monitor and troubleshoot
+them - in AI agents’ case, given its non-deterministic nature, telemetry is also
+used as a feedback loop to continuously learn from and improve the quality of
+the agent by using it as input for evaluation tools.
+
+Given both observability and evaluation tools for GenAI are from various
+vendors, it is important to establish standards around the shape of the
+telemetry generated by agent apps to avoid lock-in caused by vendor or framework
+specific formats.
+
+## The Current State of AI Agent Observability
+
+### Understanding AI Agent applications vs. AI Agent Frameworks
+
+It is crucial to distinguish between **AI Agent Application** and **AI Agent
+Frameworks**:
+
+- **AI Agent applications** refer to individual AI-driven entities that perform
+  specific tasks autonomously.
+- **AI Agent Frameworks** provide the necessary infrastructure to develop,
+  manage, and deploy AI Agents often in a more streamlined way than building an
+  agent from scratch. Examples include
+  [IBM Bee AI](https://github.com/i-am-bee),
+  [IBM wxFlow](https://github.com/IBM/wxflows/),
+  [CrewAI](https://www.crewai.com/),
+  [AutoGen](https://microsoft.github.io/autogen/dev//index.html),
+  [Semantic Kernel](https://devblogs.microsoft.com/semantic-kernel/microsofts-agentic-ai-frameworks-autogen-and-semantic-kernel/),
+  [LangGraph](https://www.langchain.com/langgraph),
+  [PydanticAI](https://ai.pydantic.dev/) and more.
+
+### Establishing a Standardized Semantic Convention
```

Comment:
Could you add a paragraph or diagram explaining the difference between the "model", "agent" and "framework" layers? As it stands, there are a lot of words and links but not much intuition for a reader who is unfamiliar with the space.

E.g., could you take a sample conversation and call out which semconvs would describe which aspects of the telemetry?